\documentclass{report}
\usepackage{color,soul,amsmath,graphicx}
\providecommand{\abs}[1]{\lvert#1\rvert}
\providecommand{\norm}[1]{\lVert#1\rVert}

\title{Exploring the quantitative dependence of proteome composition on growth rate in model microorganisms}
\author{Uri Barenholz}
\date{March 2013}

\begin{document}

\maketitle
\tableofcontents

\section{Introduction}
Single cell organisms are probably the simplest form of life accessible to science today.
However, complete understanding of their workings is still far from reached.
The ability of such organisms to self-replicate given a constant supply of simple substances is one of the simplest experimental setups that can be used to investigate the workings of a living system, albeit under a relatively simple environmental challenge.
The reasoning behind this experimental setting is that, at the absence of external changes to the environment, an organism should “converge” to a constant state (or replication cycle).
When observing a culture that grows in this condition (termed exponential balanced growth), the growth rate remains constant and the relative (average) amounts of intracellular components are also maintained \cite{Campbell1957}.
Such setups have been used for decades \cite{Schaechter1958,Maaloe1969,Pedersen1978a} to explore the details of metabolism, resource allocation and cellular composition of microorganisms under such balanced, exponential growth.
We note that the definition of exponential balanced growth is somewhat more restrictive than the often quoted ``(mid) exponential growth'' as it implies not only constant growth rate but also semi-constant cell composition.

Advances in technology allowed for continuous increase in the resolution and breadth of measurements of the different components of cells under constant conditions starting from the mere growth rate and wet/dry mass ratio, moving onto composition in terms of chemical elements, macromolecular composition and in recent years mRNA profiling and complete proteomic data using various methods.

As measurement techniques evolve and more details are exposed, models and formulas of finer detail are being devised to explain and predict various physiological aspects of growing cells \cite{Scott2010a,Lerman2012,Karr2012}.
\subsection{Research motivation}
Discovering and understanding the rules underlying the composition of cells is valuable in many ways.
It therefore has a long scientific heritage (\cite{Schaechter1958, Maaloe1969, Bremer1987, Klumpp2009a}, just to name a few).
From a basic-science point of view it increases our understanding of how microorganisms work, what are the mechanisms at their disposal and how they regulate their composition to address different environmental conditions.
Experimentally, the understanding of the impact of global, growth-rate induced, parameters on expression patterns helps in analyzing experimental results and discerning between specific regulation and global effects \cite{loven2012}.
For synthetic biology applications, modelling of the interplay between protein expression and growth rate helps in analyzing the expected properties and impacts of introducing new pathways, or modifying existing traits of target microorganisms.
We therefore find the pursue for the discovery of ``Growth Laws'' (\cite{Scott2010a}) and underlying principles to be a valuable scientific goal.
\subsection{Recent results}
In recent years there has been renewed intereset in the interplay between global factors, growth rate and gene expression.

In \cite{Zaslaver2009a} the authors monitored the activity of a large set of promoters across different growth media and different growth phases in E.Coli.
They found that the fraction of ribosomal expression out of the entire set of expressed genes grows linearly with growth rate.
Furthermore, they found that the overall distribution of promoter activities remains relatively constant but specific (metabolic) promoters present different activities depending on the growth media provided.
The authors present a model explaining the observed results for ribosomal promoters under a framework of optimal resource allocation.
According to the model, optimal allocation of resources between synthesizing ribosomal proteins and metabolic proteins in order to maximize growth rate, results in a ribosomal synthesis fraction that is linear in growth rate.
This result agrees well with their observed data.

In \cite{Levy2009} the authors examined the temporal relation between gene expression and growth rate in S.Cerevisiae.
They reason that if growth rate changes precede gene expression changes then gene expression is regulated by growth rate (termed 'feedback' regulation).
On the other hand, if gene expression changes precede growth rate changes, then sensing of the environment effects gene expression that, in turn, changes the growth rate (termed 'feed-forward' regulation).
The authors use two main experimental setups.
In one, shifts were made to the composition of the media in which the cells grew.
In the other, mutant strains lacking the ability to ferment glucose where grown in glucose rich media and their gene expression pattern was monitored.
In both cases the observed results suggest that the feed-forward mechanism was at play.
The authors thus conclude that, according to their results, carful tuning of sensing of environmental conditions regulates gene expression that, in turn, affects growth rate.

In \cite{Klumpp2009a} the question of whether the protein concentration of a constitutively expressed gene should increase or decrease as a function of growth rate in E.Coli is analyzed.
According to the authors' findings, the concentration is expected to decrease as growth rate increases.
Their calculation makes use of measured dependence on growth rate of relevant cellular parameters: cellular copy number of the gene, transcription rate per gene copy, mRNA degradation rate, translation rate per mRNA, protein degradation rate and cell volume.
Despite the fact that the predicted number of protein molecules per cell increases with growth rate, the expected increase in cell size outweigh this effect, reversing the result when protein concentration is considered.
The model was compared, with good agreement, to measurements of lacZ, TSase and OTCase reported from the mid-late 70's.
The prediction was less accurate when protein fraction out of the proteome was considered.
The article continue to analyze the expected effects of different regulatory mechanisms and their interplay with growth rate.
The authors conclude that ``Changes in gene expression need not reflect regulation''.

In \cite{Scott2010a} the impacts of modifying the translation rate of E.Coli (via antibiotics) is analyzed.
The authors derive formulas for describing the expected effects of modification of translation rate on growth rate and protein levels.
They also induce the production of unecessary proteins and monitor the effect on growth rate to assess the impact of protein burden.
Their major results are that slower translation rate slows growth and reduces protein levels and that production of unnecessary protein reduces growth rate in a proportional way.

In \cite{Berthoumieux2013} an attempt is made to distinguish between the effects of global physiological state of the cell and specific regulation mechanisms in E.Coli.
The authors use an unregulated promoter (pRM from phage $\lambda$) as a benchmark and compare the activities of other promoters (fis, crp, acs) with it.
Data is obtained by monitoring the growth of a batch culture in 96-well plate and calculating the promoter activities throughout the growth phases (using placmids with fluorescent reporter genes).
For acs the authors conclude that specific regulation due to internal cAMP concentration has a major impact on promoter activity.
For fis and crp, the major impact on promoter activity is the global physiological state of the cell.

In \cite{loven2012} the issue of normalization of microarray measurements is discussed.
The authors claim that carful procedures need to be followed in order to correctly identify what part of the expression program changes vs. what measured changes are the result of changes in global cellular parameters.
This article demonstrates another aspect in which global physiological changes play a role in observed measurements and their analysis.

\section{Research objectives}
\begin{enumerate}
\item Develop a model that relates the expression level of an unregulated gene to the growth rate and global physiological state of the cell.
\item Assess the extent to which observed gene expression across conditions is the result of global factors vs. specific regulation.
\item Analyze the tradeoff between optimal resource allocation via specific regulation and passive allocation of resources due to global cellular conditions.
\end{enumerate}
\section{Preliminary results}
\subsection{A model for estimating the effect of resource allocation on protein accumulation rate and protein level}
When comparing promoter activity or gene expression data across different growth conditions one finds that nearly all promoters change their activity.
However - detailed examination of this change reveals that genes can be subdivided into clusters and that the changes within each cluster (between two given conditions) can be encapsulated by a single scaling factor \cite{Leeat2013}.

We find this observation to suggest that promoters and genes have some intrinsic, static, affinity for expression.
In the following section we extend the analysis of the implications of having a limited synthesis capacity, as is suggested in \cite{Leeat2013} and integrate it with the implications of the existence of such static affinities.
We postulate that a large fraction of the changes observed in promoter activity and gene expression can be explained by a passive mechanism of competition over resources.

As a start we describe a culture of cells as a biomass-generating machine.
We make the assumption that, when at balanced growth, the ratio of protein to biomass remains constant (under the same growth condition).
This ratio is known to change somewhat under different growth conditions \cite{Bremer1987}.

We use the following notation:
\begin{description}
\item[$P$] Total protein amount (in culture, mass).
\item[$B$] Total biomass amount (of culture, mass).
\item [$dP/dt$] Rate of protein production (in culture, mass/time).
\item [$dB/dt$] Rate of biomass production (in culture, mass/time).
\item [$g$] Growth rate ($time^{-1}$).
\end{description}
We note that by definition:
\[ g=\frac{dB/dt}{B}\]
A corollary of the above equation is that for any biochemical entity $Q$ whose fraction out of the biomass $r_Q$ is constant (e.g. rRNA), one gets that:
\subsubsection{Corollary}
If
\[\frac{Q}{B}=r_Q\]
is constant then:
\begin{equation}
\label{global-gr}
g=\frac{dB/dt}{B}=\frac{d(r_QB)/dt}{r_QB}=\frac{dQ/dt}{Q}
\end{equation}
Namely: The total rate of Q's production divided by the total amount of Q is equal to the growth rate as long as Q's ratio out of the total biomass is constant.
This is, in fact, the definition of balanced growth \cite{Campbell1957}.

Applying equation \ref{global-gr} to the total protein production, denoting by $r$ the ratio of total protein to biomass, one gets that:
\[g=\frac{dP/dt}{P}=\frac{dP/dt}{rB}\rightarrow\frac{dP/dt}{B}=rg\]
The last term ($\frac{dP/dt}{B}$) has an interesting biological interpretation.
It is the protein production capacity of the culture per biomass unit.

We note that, again using equation \ref{global-gr}, if one assumes that the composition of the proteome remains constant under a given growth condition then for each protein $P_i$ it holds that:
\[g=\frac{dP_i/dt}{P_i}\]
That is, the production rate of protein $i$ divided by the amount of the protein in the biomass is equal to the growth rate.

This result suggests that the observed promoter activity of $P_i$ that is usually operatively defined as (though it is understood that this includes effects of the amount of the translation machinery and not only promoter effects):
\begin{equation}
\label{pa-gr-relation}
A_i=\frac{dP_i/dt}{B}=\frac{dP_i/dt}{P_i}\frac{P_i}{B}=g\frac{P_i}{B}=rg\frac{P_i}{P}
\end{equation}
Depends, by definition, both on the growth rate and on $P_i$’s fraction out of the total biomass (or out of the total protein in the biomass).
In light of this observation we aim to derive an “intrinsic”, growth-condition independent (meaning growth-rate and protein-to-biomass ratio independent), promoter activity measure.
A measure that, once determined, can be used to calculate the observed promoter activity for a given steady growth condition by taking into account the growth rate and proteome composition under this condition.
\subsubsection{The passive allocation model}
We explore a model where, under any given growth condition, each promoter has some intrinsic ``strength'' or affinity.
We stress that promoters that are consitutively expressed have the same strength under any growth condition.
We assume this condition holds for the majority of promoters (though this assumption is irrelevant for the following analysis).
Furthermore, in our model, regulated promoters can only have a few, discrete, transcriptionally controlled ``strengths'' (On-Off and rarely some intermediate states as well).
We note that for a regulated promoter, it will have the same strength in between any two growth conditions between which its specific regulation remains unchanged.
We claim that, despite the discreteness in definition, such a model will exhibit continuous changes in the observed production rates and protein levels due to the effects of growth-rate and protein composition described above.

According to our model the cell senses its growth environment and, for those promoters that can be regulated, decides in a discrete fashion on their expression strength (out of the limited set of strengths available for each such promoter).
This follows the 'feed-forward' strategy suggested by \cite{Levy2009}.
The outcome of this process is a ``task-list'' of genes to express, where each gene has its associated, intrinsic strength.
Again - for unregulated genes this strength is constant (per gene) and for regulated genes it is one of a finite subset of possible strengths.
Biologically, this list takes the form of active/ready-to-be-transcribed promoters.
This ``task-list'' is then being processed by the transcription-translation machinery.
We assume this machinery operates at its maximal capacity, given the existing growth conditions, to express the genes.
The intrinsic strengths induce a partition function (or relative expression) of different genes with respect to one another and thus determine the final ratios between the different proteins.

Formally, we define the intrinsic activity level of promoter $i$ under condition $c$ as:
\[\Gamma^c_i=\delta_i+a^c_i\beta_i ; a^c_i\in \{a_i\}\]
where $\delta_i$ is the promoter basal level and, for each promoter with active regulation, there is a finite set of activation levels {0,...,1} out of which one,$a^c_i$, is used under condition $c$.
We note that such an intrinsic activity value, $\Gamma^c_i$, can be the combined result of RBS-sequence, RNA-polymerase affinity (that can be discretely modulated by the binding of transcription factors) or other (non-context dependent) mechanisms.

We postulate that, using equation \ref{pa-gr-relation}, the activity level of promoter $i$ under condition $c$ will be:
\begin{equation}
\label{cond-act}
A^c_i=r_cg_c\frac{\Gamma^c_i}{\norm{\vec{\delta}+\vec{a^c}I\vec{\beta}}}=r_cg_c\frac{\Gamma^c_i}{\sum_{j\in P}\Gamma^c_j}
\end{equation}
(Where the middle term uses vector notation with $I$ being the identity matrix).

An important point one needs to pay attention to here is that the composition of the proteome under condition c (namely the denominators of the middle and right hand sides) may change between different growth conditions.
However, these changes are entirely encapsulated in the condition specific $\vec{a^c}$ vector of the discrete activation levels.
One of our aims in this research is to try and test to what extent this assumption holds.
\subsubsection{Deriving an explicit expected activity level}
In order to test the proposed model we need to calculate what the expected activity level under an arbitrary condition, according to our model, should be.
The term we derived in equation \ref{cond-act} above includes the sum over the static activity levels of all promoters under the relevant condition.
Using this equation is therefore somewhat impractical as the sum needs to be determined.
We continue to show how this term can be eliminated to get an observed activity level that depends only on $\Gamma^c_i$ itself.

We make two assumptions:
\begin{enumerate}
\item The translation rate is constant.
\item There is a set of genes, $G_t$ , that produce the translation machinery, and these genes are unregulated (meaning their expression level is not under active regulation).
\end{enumerate}
Under these assumptions we denote the sum of intrinsic strengths of the unregulated, translation-machinery producing genes by:
\[\Omega=\sum_{i \in G_t}\Gamma_i=\sum_{i \in G_t}\delta_i\]
Note that, as these genes are unregulated, no condition-specific activity levels get involved in the sum.
This term is therefore constant across different growth conditions.

Next we make the following observation:
As these genes encode the translation machinery proteins then, assuming constant translation rate, there is a constant time $t_{tr}$ that this mechanism takes to translate its own genes.
For simplicity one can think about the time it takes a single ribosome to translate the equivalent of its own proteins.
Note that when a cell doubles itself, every ribosome needs to translate at least the equivalent of its own proteins plus some portion of the rest of the proteome.
Therefore, the total time it takes the translation machinery to produce a desired proteome under condition c, and thus to double the biomass, assuming constant translation rate, is:
\[\tau_c=\frac{\sum_{j\in P}\Gamma^c_j}{\Omega}t_{tr}\]
Remebering that by  definition $g=\ln(2)/\tau$ yields:
\[g_c=\ln(2)\frac{\Omega}{t_{tr}\sum_{j\in P}\Gamma^c_i}\]
Substituting this result in the activity level equation (equation \ref{cond-act}) for condition c we get that:
\begin{equation}
\label{ind-act}
A^c_i=r_c g_c \frac{\Gamma^c_i}{\sum_{j\in P}\Gamma^c_j}=r_c g_c \frac{\Gamma^c_i}{\Omega}\frac{\Omega}{\sum_{j\in P}\Gamma^c_j}=r_c g_c^2\frac{t_{tr}}{\ln(2)}\frac{\Gamma^c_i}{\Omega}
\end{equation}
Leaving us with an observed activity level that depends linearly on the protein/biomass ratio, the square of the growth rate, some condition-independent constants and a condition specific intrinsic strength of the relevant promoter, namely: $\frac{\Gamma^c_i}{\Omega}$.
\subsubsection{Accounting for variable translation rates}
As noted above, the preceding analysis assumes the translation rate (and as a result, $t_tr$) remains constant under different growth conditions.
Measurements suggest this is not necessarily the case \cite{Liang2000}.
We note that this assumption was used in exactly one place, that is, in the assertion that $t_{tr}$ is constant, as that is the only place where there is a direct connection between intrinsic strengths and translation times.
Thus, relieving this assumption and instead assuming that the translation rate is some function of the growth rate, $f$, one gets that the time the translation mechanism takes to translate itself is simply:
\begin{equation}
t_{tr}=\frac{T}{f(g_c)}
\end{equation}
Where $T$ is the size of the translation machinery in amino-acids.
Incorporating this term into the activity level equation (equation \ref{ind-act}) we conclude that in such cases:
\begin{equation}
A^c_i=r_c g_c \frac{\Gamma^c_i}{\sum_{j\in P}\Gamma^c_j}=r_c g_c \frac{\Gamma^c_i}{\Omega}\frac{\Omega}{\sum_{j\in P}\Gamma^c_j}=r_c g_c^2\frac{T}{f(g_c)\ln(2)}\frac{\Gamma^c_i}{\Omega}
\end{equation}
It is reasonable to expect the dependence of the translation rate on the growth rate to be a monotonic, slowly increasing function.
In such a case, the resulting final relation of the production rate to the growth rate should therefore be a sub-quadratic relation.
\subsubsection{Intermediate conclusions}
We have developed a model that predicts the protein level and promoter activity for unregulated genes as a function of growth rate.
Our model employs a 'feed-forward' strategy as was suggested in \cite{Levy2009}.
It takes a top-down approach and extends the analysis done in \cite{Leeat2013}.
Our model suggests that the promoter activity scales like the growth rate squared and that the protein level scales like the growth rate.
These results are the inverse of the claims made at \cite{Klumpp2009a} and \cite{Scott2010a}.
Our model claims that the relation observed for ribosomal synthesis fraction and ribosomal protein fraction out of the proteome can in fact be extended to other promoters and proteins, as long as they are assumed to be unregulated between conditions.
It thus adds aditional explenation to the explenations already given for the observed dependence of ribosomal protein synthesis and levels as a function of growth rate \cite{Zaslaver2009a}.
Our research plan is to use this model to test the hypothesis that global cellular parameters play in fact a larger role than previously appreciated in determining promoter activiy and resulting protein levels, as is claimed by \cite{Berthoumieux2013}.
\subsection{Growth rate, protein level and promoter activity follow the model when tracked over time}
One of the results of our analysis is that intrinsic strengths suggest conservation of relative levels of protein and activities of promoters.
However, protein level should scale like the growth rate, while promoter activity should scale like the growth rate squared.
To test this hypothesis, albeit at non-ideal conditions, we tracked the growth of E.Coli in a batch culture growing in M9 minimal media with 0.2\% glucose.
The strain grown had a placmid (pTAC) fused to two fluorescent reporter genes (YFP and mCherry).
At any time-point we calculated the growth rate, the promoter activity and the protein level for each of the proteins.
The results are shown in figure \ref{time-gr-fig}.
We identified two phases of relatively constant growth rate, as can be seen in the figure.
The relative abundance and accumulation rate of the two proteins are maintained in between these two phases.
However, the absolute level of the proteins scales like the growth rate whereas the accumulation rate scales like the growth rate squared.
This initial result thus validates the ability of our experimental system and analysis to quantify the relevant effects and shows one of the predictions of our model at play.
It should be noted that as the two genes where part of the same operon, this result is still lacking as it does not monitor the ratio between two different, unregulate promoters.
\begin{figure}[h]
\includegraphics{propfig1.pdf}
\caption{Comparing growth rate, protein level and promoter activity throughout growth in a batch culture. The ratio between the two proteins is maintained throughout growth. The ratio between the accumulation rate of the two proteins is maintained throughout growth and is equal to the ratio between the protein levels. Changes in growth rate and protein levels are synchronized. Changes in protein level are proportional to changes in growth rate. Changes in promoter activity precede changes in growth rate and protein level by about one hour. Changes in promoter activity are proportional to the square of the changes in growth rate.}
\label{time-gr-fig}
\end{figure}
-------------------------------------------------------------------------------
As a test case for the model and our ability to set up an experimental system sensitive enough to test it, Leeat Y. assembled a plate spanning various media with a strain of S. Cerevisiae expressing GFP fused to a ribosomal promoter.

Our analysis shows that, indeed, the accumulation rate scales like the growth rate squared.
One should note that there are other explanations for this dependence and therefore other promoters will be tested as well.
\subsection{Exploring balanced growth in batch cultures}
Although logarithmic balanced growth is well defined theoretically, it remains unclear if and how real cultures can exist in this phase.

One device that is commonly used for long term growth of cultures is a chemostat.
A comprehensive discussion of chemostats and their workings in beyond the scope of this document.
Briefly, chemostats replace a pre-defined fraction of a media in which a culture in grown by fresh media every some constant time interval.
They therefore allow continous growth over long periods of time.
However, it is unclear whether growth under such conditions is indeed balanced.
This is due to the fact that the culture converges to being under some limiting nutrient concentration.
When such a limiting concentration is reached, the growth of the culture is slowed down (prior to the next dilution step) and therefore the logarithmic balanced growth phase breaks.

Another way of apporximating balanced growth is by putting a small culture in a large volume of media, a situation that, for some time, allows the culture to grow with no limiting resource.
One of my aims in this research is to quantify the extent to which logarithmic balanced growth occures in such a setup.
The concrete experimental setup consists of a 96-well plate and a plate reader for measuring both the optical density and fluorescense of plasmids that are integrated into the bacteria.

Being able to identify logarithmic balanced growth given such a setup (if indeed it occures) is a valuable building block for extracting various quantitative data about different bacterial strains under different nutrients.
It should allow for good reproducibility and insightful comparisons of phenotipic behaviours that are hard to obtain otherwise.
\subsubsection{Growth rate assessment}
\label{growth-rate}
A key feature of logarithmic balanced growth is the maintenance of growth rate throughout this phase.
As such, accurate assessment of growth rate is required.

Despite being theoretically simple to calculate, in practice, obtaining accurate growth rate assessment over time requires overcoming various experimental obstacles.
These include, among others: accurate background determination and subtraction, detector noise, outlying measurements detection and removal and the detection of the period of time during which the growth rate is constant and maximal.

In our work we have developed a procedure that attempts to overcome these issues.
The key features of this procedure include background and detector noise determination and subtraction, use of high dilution ratio of the initial culture and use of Trimmed-Least-Squares robust fit methods \hl{cite reuseeaue} to overcome outlier measurements \hl{figure 1 - 4 panes - raw, background and threshold subtraction, trimming and log-scale, growth rate}.

We believe the combination of these techniques to give us a reliable, reproducable and versatile growth rate assessment method for a varaiety of growth media and organisms.

Where fluorescent reporter genes exist, we can estimate the accuracy of the measurement by comparing the results of executing this algorithm on the OD measurements with those obtained by executing it on fluorescense measurements.
If deviations between the two estimates occure we can conclude that either the growth is not balanced, or the procedure failed.

Further verification that the growth is indeed balanced during a phase can be done by verifying no correlation between the growth rate at a sliding subwindow throughout this time and the time itself.
\subsubsection{Protein level and promoter activity measurements}
Two features that are of interest for modeling the proteome of a cell, be it during balanced growth or otherwise, are the relative protein levels and the corresponding promoter activities.
Both features are hard, sometimes impossible, to measure directly without bias.

The fusion of fluorescent reporter genes to promoters in libraries is a common method to quantitatively deduce these features.
However, there is no canonical form or protocol for the inference of these features based on such experimental setup.
Two leading approaches involve estimating the amount of fluorescence per OD unit, which corresponds to protein abundance, and measuring the rate at which the fluorescence increases per OD unit, which corresponds to the production (or, more accurately, accumulation) rate of the protein.

While these two values are obtained from the same measurements, they are by no means equivalent.
If one assumes the protein level to follow the classic formula:
\[ \frac{dP}{dt}=\beta-\alpha P\]
(Where $\alpha$ is the dilution rate, equivalent to the growth rate, and $\beta$ is the production rate)
Then the steady-state concentration of the protein becomes:
\[ P_{st.st}=\frac{\beta}{\alpha}\]
Showing that the steady state concentration equales the production rate divided by the growth rate.

Monitoring these two properties throughout the growth phase of a culture gives us an opportunity to estimate how well the above stated mathematical relation holds and allows us to test if and how the expected convergence occures \hl{figure 2. 4 panels - fluorescence and OD growth rates on the same plot, activity over time, abundance over time and ratio of abundance to activity over time compared with growth rate}.
In order to achieve these goals we are developing procedures that allow us to extract accurate estimates for the fluorescence and OD values throughtout the growth phase.

Briefly, our procedure involves calculating linear fit on the background-subtracted logarithm of both the fluorescence and abosrbance measurements (in a similar manner to the one described in \ref{growth-rate}).
Comparison of the calculated slopes indicates how balanced the protein accumulation is compared with the growh rate.
The ratio between the calculated offsets of the fit lines measures the ratio of fluorescence to biomass which serves as a proxy for the protein abundance.
\subsection{Correlating growth rate and the protein to biomass ratio}
A simple calculation shows that the protein to biomass ratio is expected to change as a function of the growth rate.
As Ribosomes are composed of 1:2 ratio of protein to RNA then, under the assumption of constant translation rate, one expects that the ribosomal protein fraction out of the proteome will be:
\[\alpha_R=\frac{t_{tr}}{\tau}=t_{tr}g/\ln(2)\]
And therefore, the ribosomal RNA (rRNA) amount relative to the protein amount should follow:
\[\alpha_{rRNA}=2\alpha_R=2t_{tr}g/\ln(2)\]
Meaning it is an increasing function of $g$.
Experimental data supports this model as it shows increase in RNA/protein ratio as growth rate increases.
Note that the other factors needed for calculation of the global macromolecular composition are the relative fractions of other cellular components (lipids, DNA, small metabolites, all of which are expected to stay relatively constant across different growth rates) and the relative fraction of rRNA out of the total RNA in the cell (which is commonly cited as being ~ 85\% as long as growth rates are not extremely slow \cite{Bremer1987,goldberger1979biological}).

We aim to measure the RNA to protein ratio to check the validity of this prediction.
\bibliographystyle{plain}
\bibliography{library}
\end{document}
