\documentclass[a4page,notitlepage]{article}
\usepackage{color,soul,amsmath,graphicx}
\providecommand{\abs}[1]{\lvert#1\rvert}
\providecommand{\norm}[1]{\lVert#1\rVert}

\title{Exploring the quantitative dependence of proteome composition and growth rate in model microorganisms}
\author{Uri Barenholz}
\date{March 2013}

\begin{document}

\maketitle
\begin{abstract}
Exploring the relation between growth rate and different cellular parameters in microorganisms has been a research subject for many decades.
Recent results suggest this relation extends beyond global parameters such as size, protein to RNA ratio and ribosomes concentration.
In this work we attempt to characterize the interdependence of growth rate and specific promoter activities, focusing on non-regulated promoters.
For non-regulated promoters we suggest a ``passive'' regulation model where observed changes in promoter activity are the passive outcome of global changes in the cell physiology and expression program.
Our initial results suggest an expected linear dependence of protein level to growth rate and a scaling of promoter activity that is proportional to the growth rate squared for such promoters.
\end{abstract}
\clearpage
\tableofcontents

\section{Introduction}
Single cell organisms are probably the simplest form of life accessible to science today.
However, complete understanding of their workings is still far from reached.
Decades ago it was first observed that, under constant supply of nutrients, such organisms maintain a constant growth rate, as was summarized by \cite{Monod1949}.
A decade later is was discovered that growth rate is a major determinant of various cellular properties such as cell size and macromolecular composition, while growth media seems to play relatively minor role in these aspects \cite{Schaechter1958}.
Complementary research suggested that, under constant growth conditions, microorganisms should ``converge'' to a constant state (or replication cycle) termed ``exponential balanced growth'' \cite{Campbell1957}.
This state is characterized, on top of a constant growth rate, by a constant (average) ratio of intracellular components.
Throughout the years there has been a continuous interest in the interplay between growth rate and cellular composition (\cite{Maaloe1969,Pedersen1978a,Bremer1987}).
However, it was not until recent years when advances in technology enabled high resolution monitoring of cellular properties such as mRNA levels and wide proteomic data.
Despite these advances, until recently, the connection between changes in individual transcripts and proteins and cellular physiology, as reflected by growth rate, remained largely unaddressed.

In recent years, there has been renewed interest in the relationship between growth rate, gene expression and protein abundance, and attempts have been made to characterize it experimentally (\cite{Zaslaver2009a,Levy2009,Berthoumieux2013}) and theoretically (\cite{Klumpp2009a,Scott2010b}).

Key results from recent studies indicate that cells respond to specific cues from the environment to control their proteome composition and that this composition, in turn, influences growth rate (\cite{Levy2009}).
However, despite the differences in proteomic composition, the distribution of activities among proteomic functions seems to be relatively constant (\cite{Zaslaver2009a}).
In a somewhat contradictory fashion, \cite{Berthoumieux2013} suggests that global cellular effects are major determinants of expression level and that specific regulation plays a relatively minor role in fine-tuning expression.
\cite{loven2012} notes that global effects can have non-trivial implications on data analysis and that special care should be taken when dealing with expression data obtained under different growth conditions.
Recent theoretical analyses (\cite{Klumpp2009a,Scott2010b}) suggest non-trivial dependence of constitutively expressed promoters on growth rate.
Such theories and insights thus help augmenting our understanding of resource allocation and trade-offs governing cellular composition.

\subsection{Research motivation}
Discovering and understanding the rules underlying the composition of cells is valuable in many ways.
It therefore has a long scientific heritage (\cite{Schaechter1958, Maaloe1969, Bremer1987, Klumpp2009a}, just to name a few).
From a basic-science point of view it increases our understanding of how microorganisms work, what are the mechanisms at their disposal and how they regulate their composition to address different environmental conditions.
Experimentally, understanding the impact of global, passive, growth-rate induced parameters on expression patterns helps in analyzing experimental results and discerning between specific regulation and global effects \cite{loven2012}.
For synthetic biology applications, modeling of the interplay between protein expression and growth rate helps in analyzing the expected properties and impacts of introducing new pathways, or modifying existing traits of target microorganisms.
It also helps in forming in silico whole-cell models such as \cite{Lerman2012} and \cite{Karr2012} and relating their results to measured values.
We therefore find the pursue for the discovery of ``Growth Laws'' (\cite{Scott2010b}) and underlying principles to be a valuable scientific goal.

\subsection{Recent published results}
\hl{Following Leeat's suggestions I've added references and one-sentence summaries to these articles to the general introduction section. This section can therefore be dropped altogether. Please advise if you think I should keep it as a more detailed survey or not.}

In recent years there has been renewed interest in the interplay between global factors, growth rate and gene expression.

In \cite{Zaslaver2009a} the authors monitored the activity of a large set of promoters across different growth media and different growth phases in \emph{E.Coli}.
They found that the fraction of ribosomal expression out of the entire set of expressed genes grows linearly with growth rate.
Furthermore, they found that the overall distribution of promoter activities remains relatively constant but specific (metabolic) promoters present different activities depending on the growth media provided.
The authors present a model explaining the observed results for ribosomal promoters under a framework of optimal resource allocation.
According to the model, optimal allocation of resources between synthesizing ribosomal proteins and metabolic proteins in order to maximize growth rate, results in a ribosomal synthesis fraction that is linear in growth rate.
This result agrees well with their observed data.

In \cite{Levy2009} the authors examined the temporal relation between gene expression and growth rate in \emph{S.Cerevisiae}.
They reason that if growth rate changes precede gene expression changes then gene expression is regulated by growth rate (termed 'feedback' regulation).
On the other hand, if gene expression changes precede growth rate changes, then sensing of the environment effects gene expression that, in turn, changes the growth rate (termed 'feed-forward' regulation).
The authors use two main experimental setups.
In one, shifts were made to the composition of the media in which the cells grew.
In the other, mutant strains lacking the ability to ferment glucose where grown in glucose rich media and their gene expression pattern was monitored.
In both cases the observed results suggest that the feed-forward mechanism was at play.
The authors thus conclude that, according to their results, careful tuning of sensing of environmental conditions regulates gene expression that, in turn, affects growth rate.

In \cite{Klumpp2009a} the question of whether the protein concentration of a constitutively expressed gene should increase or decrease as a function of growth rate in \emph{E.Coli} is analyzed.
According to the authors' findings, the concentration is expected to decrease as growth rate increases.
Their calculation makes use of measured dependence on growth rate of relevant cellular parameters: cellular copy number of the gene, transcription rate per gene copy, mRNA degradation rate, translation rate per mRNA, protein degradation rate and cell volume.
Despite the fact that the predicted number of protein molecules per cell increases with growth rate, the expected increase in cell size outweigh this effect, reversing the result when protein concentration is considered.
The model was compared, with good agreement, to measurements of lacZ, TSase and OTCase reported from the mid-late 70's.
The prediction was less accurate when protein fraction out of the proteome was considered.
The article continue to analyze the expected effects of different regulatory mechanisms and their interplay with growth rate.
The authors conclude that ``Changes in gene expression need not reflect regulation''.

In \cite{Scott2010b} the impacts of modifying the translation rate of \emph{E.Coli} (via antibiotics) is analyzed.
The authors derive formulas for describing the expected effects of modification of translation rate on growth rate and protein levels.
They also induce the production of unnecessary proteins and monitor the effect on growth rate to assess the impact of protein burden.
Their major results are that slower translation rate slows growth and reduces protein levels and that production of unnecessary protein reduces growth rate in a proportional way.

In \cite{Berthoumieux2013} an attempt is made to distinguish between the effects of global physiological state of the cell and specific regulation mechanisms in \emph{E.Coli}.
The authors use an unregulated promoter (pRM from phage $\lambda$) as a benchmark and compare the activities of other promoters (fis, crp, acs) with it.
Data is obtained by monitoring the growth of a batch culture in 96-well plate and calculating the promoter activities throughout the growth phases (using plasmids with fluorescent reporter genes).
For acs the authors conclude that specific regulation due to internal cAMP concentration has a major impact on promoter activity.
For fis and crp, the major impact on promoter activity is the global physiological state of the cell.

In \cite{loven2012} the issue of normalization of micro-array measurements is discussed.
The authors claim that careful procedures need to be followed in order to correctly identify what part of the expression program changes vs. what measured changes are the result of changes in global cellular parameters.
This article demonstrates another aspect in which global physiological changes play a role in observed measurements and their analysis.

\section{Research objectives}
\begin{enumerate}
\item Develop a model that relates the expression level of an unregulated gene to the growth rate and global physiological state of the cell.
\item Assess the extent to which observed gene expression across conditions is the result of global factors vs. specific regulation.
\item Analyze the trade-off between optimal resource allocation via specific regulation and passive allocation of resources due to global cellular conditions.
\end{enumerate}
\section{Preliminary theoretical results}
\subsection{A model for estimating the effect of resource allocation on protein accumulation rate and protein level}
When comparing promoter activity or gene expression data across different growth conditions one finds that nearly all promoters change their activity.
However - detailed examination of this change reveals that genes can be subdivided into clusters and that the changes within each cluster (between two given conditions) can be encapsulated by a single scaling factor \cite{Leeat2013}.

We find this observation to suggest that promoters and genes have some intrinsic, static, affinity for expression, as was first suggested decades ago by \cite{Maaloe1969}.
In the following section we extend the analysis of the implications of having a limited synthesis capacity, as is suggested in \cite{Leeat2013} and integrate it with the implications of the existence of such static affinities.
We postulate that a large fraction of the changes observed in promoter activity and gene expression can be explained by a passive mechanism of competition over resources.

As a start we describe a culture of cells as a biomass-generating machine.
We follow the definition of balanced growth from \cite{Campbell1957} according to which, when at this state, every extensive property of the system grows at the same rate.
Our analysis is based on the assumption that the observed system is in exponential, balanced growth.

We use the following notation:
\begin{description}
\item[$P$] Total protein amount (of culture, mass).
\item[$B$] Total biomass amount (of culture, mass).
\item [$dP/dt$] Rate of protein production (in culture, mass/time).
\item [$dB/dt$] Rate of biomass production (in culture, mass/time).
\item [$g$] Growth rate ($time^{-1}$) such that $B(t)=B_0e^{gt}$.
\end{description}
We note that by definition:
\[ g=\frac{dB/dt}{B}\]
A corollary of the above equation is that for any biochemical entity $x$ whose fraction out of the biomass $r_x$ is constant (e.g. rRNA), one gets that:
\subsubsection{Corollary}
If
\[\frac{x}{B}=r_x\]
is constant then:
\begin{equation}
\label{global-gr}
g=\frac{dB/dt}{B}=\frac{d(r_xB)/dt}{r_xB}=\frac{dx/dt}{x}
\end{equation}
Namely: The total rate of x's production divided by the total amount of x is equal to the growth rate as long as x's ratio out of the total biomass is constant.
This is a mathematical restatement of the definition of balanced growth from \cite{Campbell1957}.

Applying equation \ref{global-gr} to the total protein production, denoting by $r$ the ratio of total protein to biomass, one gets that:
\[g=\frac{dP/dt}{P}=\frac{dP/dt}{rB}\rightarrow\frac{dP/dt}{B}=rg\]
The last term ($\frac{dP/dt}{B}$) has an interesting biological interpretation.
It is the protein production capacity of the culture per biomass unit.
Note that, while $r$ is expected to be constant under any specific condition, it is known to change somewhat between different growth conditions as was observed by \cite{Bremer1987}

We note that, again using equation \ref{global-gr}, if one assumes that the composition of the proteome remains constant under a given growth condition then for each protein $P_i$ it holds that:
\[g=\frac{dP_i/dt}{P_i}\]
That is, the accumulation rate of protein $i$ divided by the amount of the protein in the biomass is equal to the growth rate.

This result suggests that the observed promoter activity of $P_i$, that is usually operatively defined as (though it is understood that this includes effects of the amount of the translation machinery and not only promoter effects):
\begin{equation}
\label{pa-gr-relation}
A_i=\frac{dP_i/dt}{B}=\frac{dP_i/dt}{P_i}\frac{P_i}{B}=g\frac{P_i}{B}=rg\frac{P_i}{P}
\end{equation}
Depends, by definition, both on the growth rate and on $P_i$’s fraction out of the total biomass (or out of the total protein in the biomass).
In light of this observation we aim to derive an “intrinsic”, growth-condition independent (meaning growth-rate and protein-to-biomass ratio independent), promoter activity measure.
A measure that, once determined, can be used to calculate the observed promoter activity for a given steady growth condition by taking into account the growth rate and proteome composition under this condition.
\subsubsection{The passive allocation model}
We explore a model where, under any given growth condition, each promoter has some intrinsic ``strength'' or affinity.
We stress that promoters that are constitutively expressed have the same intrinsic strength under any growth condition.
We assume this ``constitutive state'' holds for the major part of active promoters (though this assumption is irrelevant for our analysis).
Furthermore, in our model, regulated promoters can only have a few, discrete, transcriptionally controlled, intrinsic strengths (On-Off and rarely some intermediate states as well).
For example, in our model, the Gal operon will have two sates: ``on'', with some intrinsic strength, when Galactose is present in the medium at sufficiently high concentration, and ``off'', with some intrinsic lower strength, when Galactose is absent from the medium.
Notably, in our model, the Gal operon has only two intrinsic strengths and it does not span a continuum of activation strengths.
We therefore state that for a regulated promoter, it will have the same intrinsic strength in between any two growth conditions between which its specific regulation remains unchanged.
We claim that despite the discreteness in definition, such a simplified model will exhibit continuous changes in the observed activity and protein levels due to the effects of growth-rate and protein composition described above.

According to our model the cell senses its growth environment and, for those promoters that can be regulated, decides in a discrete fashion on their expression strength (out of the limited set of strengths available for each such promoter).
Practically, this can be done by activating/deactivating transcription factors that interact with relevant sets of promoters to select their relevant intrinsic strength.
This follows the 'feed-forward' mechanism suggested by \cite{Levy2009}.
The outcome of this process is a ``task-list'' of genes to express, where each gene has its associated, intrinsic strength.
Again - for unregulated genes this strength is constant (per gene) and for regulated genes it is one of a finite subset of possible strengths.
Biologically, this list takes the form of active/ready-to-be-transcribed promoters.
This ``task-list'' is then being processed by the transcription-translation machinery.
We assume this machinery operates at its maximal capacity, given the existing growth conditions, to express the genes.
The intrinsic strengths induce a partition function (or relative expression) of different genes with respect to one another and thus determine the final ratios between the different proteins.

Formally, we define the intrinsic strength, or activity, of promoter $i$ under condition $c$ as:
\[\Gamma^c_i=\delta_i+a^c_i\beta_i ; a^c_i\in \{a_i\}\]
where $\delta_i$ is the promoter basal level and, for each promoter with active regulation, there is a maximal activation level of $\delta_i+\beta_i$ and a finite set of relative activation levels {0,...,1} out of which one,$a^c_i$, is used under condition $c$.
We note that such an intrinsic strength value, $\Gamma^c_i$, can be the combined result of RBS-sequence, RNA-polymerase affinity (that can be discretely modulated by the binding of transcription factors) or other (non-context dependent) mechanisms.

We postulate that, using equation \ref{pa-gr-relation}, the measured activity level of promoter $i$ under condition $c$ will be:
\begin{equation}
\label{cond-act}
A^c_i=r_cg_c\frac{\Gamma^c_i}{\norm{\vec{\delta}+\vec{a^c}I\vec{\beta}}}=r_cg_c\frac{\Gamma^c_i}{\sum_{j\in P}\Gamma^c_j}
\end{equation}
(Where the middle term uses vector notation with $I$ being the identity matrix).

An important point one needs to pay attention to here is that the composition of the proteome under condition c (namely the denominators of the middle and right hand sides) may change between different growth conditions.
However, these changes are entirely encapsulated in the condition specific $\vec{a^c}$ vector of the discrete activation levels.
One of our aims in this research is to try and test to what extent this assumption holds.
\subsubsection{Deriving an explicit expected activity level}
In order to test the proposed model we need to calculate what the expected activity level under an arbitrary condition, according to our model, should be.
The term we derived in equation \ref{cond-act} above includes the sum over the static activity levels of all promoters under the relevant condition.
Using this equation is therefore somewhat impractical as the sum needs to be determined.
We continue to show how this term can be eliminated to get an observed activity level that depends only on $\Gamma^c_i$ itself.

We make two assumptions:
\begin{enumerate}
\item The translation rate is constant across conditions (see for example \cite{Neidhardt1999a}).
This assumption can be relieved as is discussed in section \ref{nonconst-trans}.
\item There is a set of genes, $G_t$ , that produce the translation machinery, and these genes are unregulated (meaning their expression level is not under active regulation).
\end{enumerate}
Under these assumptions we denote the sum of intrinsic strengths of the unregulated, translation-machinery producing genes by:
\[\Omega=\sum_{i \in G_t}\Gamma_i=\sum_{i \in G_t}\delta_i\]
Note that, as these genes are unregulated, no condition-specific activity levels get involved in the sum.
This term is therefore constant across different growth conditions.

Next we make the following observation:
As these genes encode the translation machinery proteins then, assuming constant translation rate, there is a constant time $t_{tr}$ that this mechanism takes to translate its own genes.
For simplicity one can think about the time it takes a single ribosome to translate the equivalent of its own proteins.
Note that when a cell doubles itself, every ribosome needs to translate at least the equivalent of its own proteins plus some portion of the rest of the proteome.
Therefore, the total time it takes the translation machinery to produce a desired proteome under condition c, and thus to double the biomass, assuming constant translation rate, is:
\[\tau_c=\frac{\sum_{j\in P}\Gamma^c_j}{\Omega}t_{tr}\]
Remembering that by  definition $g=\ln(2)/\tau$ yields:
\[g_c=\ln(2)\frac{\Omega}{t_{tr}\sum_{j\in P}\Gamma^c_i}\]
Substituting this result in the activity level equation (equation \ref{cond-act}) for condition c we get that:
\begin{equation}
\label{ind-act}
A^c_i=r_c g_c \frac{\Gamma^c_i}{\sum_{j\in P}\Gamma^c_j}=r_c g_c \frac{\Gamma^c_i}{\Omega}\frac{\Omega}{\sum_{j\in P}\Gamma^c_j}=r_c g_c^2\frac{t_{tr}}{\ln(2)}\frac{\Gamma^c_i}{\Omega}
\end{equation}
So the observed activity level depends linearly on the protein/biomass ratio ($r_c$), the square of the growth rate ($g_c$), some condition-independent constants ($t_{tr}$, $ln(2)$ and $\Omega$) and a condition specific intrinsic strength of the relevant promoter, namely: ${\Gamma^c_i}$.
For simplicity, intrinsic strengths can be normalized to $\Omega$ so that $\Gamma_i^{c'} = \frac{\Gamma^c_i}{\Omega}$.
\subsubsection{Accounting for variable translation rates}
\label{nonconst-trans}
As noted above, the preceding analysis assumes the translation rate (and as a result, $t_{tr}$) remains constant under different growth conditions.
Measurements suggest this is not necessarily the case \cite{Liang2000}.
We note that this assumption was used in one place, in the assertion that $t_{tr}$ is constant, as that is the only place where there is a direct connection between intrinsic strengths and translation times.
Thus, relieving this assumption and instead assuming that the translation rate is some function of the growth rate, $f(g_c)$, one gets that the time the translation mechanism takes to translate itself is simply:
\begin{equation}
t_{tr}=\frac{T}{f(g_c)}
\end{equation}
Where $T$ is the size of the translation machinery in amino-acids.
Incorporating this term into the activity level equation (equation \ref{ind-act}) we conclude that in such cases:
\begin{equation}
A^c_i=r_c g_c \frac{\Gamma^c_i}{\sum_{j\in P}\Gamma^c_j}=r_c g_c \frac{\Gamma^c_i}{\Omega}\frac{\Omega}{\sum_{j\in P}\Gamma^c_j}=r_c g_c^2\frac{T}{f(g_c)\ln(2)}\frac{\Gamma^c_i}{\Omega}
\end{equation}
It is reasonable to expect the dependence of the translation rate on the growth rate to be a monotonic, slowly increasing function.
Therefore, the resulting final relation of the activity level to the growth rate should be a sub-quadratic relation.
\subsubsection{Intermediate conclusions}
We have developed a model that predicts the protein level and promoter activity for unregulated genes as a function of growth rate.
Our model employs a 'feed-forward' strategy as was suggested in \cite{Levy2009}.
It takes a top-down approach integrating global constraints and thus extends the analysis done in \cite{Leeat2013} and \cite{Maaloe1969} to include explicit dependence on growth rate.
Our model suggests that the promoter activity, given that no specific regulation is at play, scales like the growth rate squared and that the protein level scales like the growth rate.
These results are contradictory to the results stated by \cite{Klumpp2009a} and \cite{Scott2010b}, who suggest that the level of non-regulated proteins decrease with growth rate.
Our model is supported by observations that the ribosomal synthesis rate scales quadratically with growth rate (\cite{Zaslaver2009a}) and that ribosomal protein fraction scales linearly with growth rate (\cite{Maaloe1969,ingraham1983growth}).
Moreover, it implies that these relations extend beyond the well-studied ribosomal proteins to all non-regulated proteins across all growth conditions.
These relations should also apply to regulated genes across conditions in which they are not differentially regulated.
It thus adds additional explanation to the mechanisms already given for the observed dependence of ribosomal protein synthesis and levels as a function of growth rate \cite{Zaslaver2009a}.
Our research plan is to use this model to test the hypothesis that global cellular parameters play in fact a larger role than previously appreciated in determining promoter activity and resulting protein levels, as is claimed by \cite{Berthoumieux2013,Leeat2013,loven2012,Scott2010b,Klumpp2009a} and to add a new quantitative assessment to this effect.
\section{Preliminary experimental results}
\subsection{Growth rate, protein level and promoter activity follow model predictions when tracked over time}
One of the results of our analysis is that intrinsic strengths suggest conservation of relative levels of protein and activities of promoters.
However, protein level should scale like the growth rate, while promoter activity should scale like the growth rate squared.
To test this hypothesis, we tracked the growth of \emph{E.Coli} in a batch culture growing in M9 minimal media with 0.2\% glucose.
The strain grown had a plasmid (pTAC) containing two fluorescent reporter genes (YFP and mCherry).
At any time-point we calculated the growth rate, the protein accumulation rate (that serves as a proxy for the promoter activity) and the protein level for each of the proteins.
The results are shown in figure \ref{time-gr-fig}.
We identified two phases of relatively constant growth rate, fast growth and slow growth, as can be seen in the figure.
The relative abundance and accumulation rate of the two proteins are maintained in between these two phases.
However, the absolute level of the proteins scales like the growth rate whereas the accumulation rate scales like the growth rate squared.
This initial result thus validates the ability of our experimental system and analysis to quantify the relevant effects and shows one of the predictions of our model at play.
It should be noted that as the two genes where part of the same operon, this result is still lacking as it does not monitor the ratio between two different, unregulated promoters.
\begin{figure}[h]
\includegraphics{propfig1.pdf}
\caption{Inter-dependence of growth rate, protein level and protein accumulation rate follow theoretical model predictions when tracked over time in a batch culture.
All values where calculated by averaging over 18 measurement point windows spanning 2.5 hours.
(A) Growth rate as a function of time.
Two phases of relatively constant growth are marked for convenience - Maximal growth and slow growth.
The growth rate changes by a factor of about 2/3 between the two phases.
(B) Protein level per OD as a function of time.
The ratio between the levels of the two proteins remains relatively constant and is approximately 2:1.
The absolute level per OD of each protein decreases during the same time, and in the same proportion as the growth rate.
(C) Protein accumulation rate per OD as a function of time.
Changes in the accumulation rate per OD (reflecting changes in promoter activity) precede changes in growth rate and protein level by about 1.5 hours.
The ratio between the accumulation rate of the two proteins remains relatively constant and is approximately 2:1
Changes in accumulation rate per OD are proportional to the square of the changes in growth rate ($2/3^2=4/9)$.}
\label{time-gr-fig}
\end{figure}
\subsection{Promoter activity scales superlinearly with growth rate for some non-ribosomal promoters}
In an initial attempt to test our model's predictions we conducted an experiment in which seven strains of \emph{S.Cerevisiae}, each one containing a different promoter fused to a fluorescent reporter gene, were grown in a 96-well plate with the optical density and fluorescence monitored over time.
The strains were grown in six different media, differing in carbon source and availability of amino acids.
For each strain the maximal growth rate, promoter activity and protein level (during the maximal growth rate time) where calculated.
The results for one of the promoters (PAB1, mRNA polyA binding protein) is plotted in figure \ref{gr-fl-fig}.
It should be noted that the data analysis is still preliminary and quality control measures, as well as more adequate methods for identifying trends should be employed in the data processing procedure.
However, even under the initial analysis done, promoter activity shows a superlinear relation to growth rate, whereas protein level follows a more linear trend.
Figure \ref{gr-fl-mult} shows the data for the other promoters monitored.
\begin{figure}[h]
\includegraphics{propfig2.pdf}
\caption{Dependence of promoter activity and protein level on growth rate for PAB1 promoter under different growth media.
(A) Promoter activity (as deduced by protein accumulation rate) exhibits a superlinear relation to growth rate.
Dashed line shows best square fit, as the model suggests.
(B) Protein level scales roughly like growth rate.
Dashed line shows best linear fit, as the model suggests.
}
\label{gr-fl-fig}
\end{figure}
\begin{figure}[h]
\includegraphics{propfig3.pdf}
\caption{Dependence of promoter activity and protein level on growth rate under different growth media for various promoters.
Left column shows promoter activity (as deduced from protein accumulation rates).
Right column shows protein levels.
Dashed lines in left/right columns show best square/linear fit respectively, as the model suggests.
}
\label{gr-fl-mult}
\end{figure}
\section{Methods}
Promoter activity and protein level will be estimated based on promoter-reporter fusion libraries.
Such libraries exist for \emph{E.Coli} (in the Alon lab) and for \emph{S.Cerevisiae} (in the Segal lab).

Growth rate will be modified by use of different carbon sources and addition/omission of amino acids to the media.
Current planned carbon sources for \emph{S.Cerevisiae} are Glucose, Fructose, Galactose, Glycerol and Ethanol.
For \emph{E.Coli} the carbon sources are still to be determined.
One key requirement for the selected growth media is spanning of a relatively wide range of growth rates in a relatively continuous manner.

Measurements will take place by tracking optical density and fluorescence in 96-well plates using a spectrophotometer.
Analysis and extraction of relevant parameters (identification of maximal growth-rate phase, calculation of promoter activity and protein level) will be done by custom developed algorithms.
\bibliographystyle{ieeetr}
\bibliography{library}
\end{document}
